ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -132,6 +131,21 @@ tests:
     - ref: stackrox-e2e
     workflow: ipi-aws
   timeout: 5h0m0s
+- always_run: false
```

Comment:
I did not know this could be added to the `config` side and propagate to the `jobs`.

Reply:
No it does not propagate.
Changing a value of `always_run` in `config` does not change the value in `jobs` after `make update`.

It is coincidental that, if you set `always_run: false` in `config`, then delete current `always_run` in `jobs` and re-run `make update`, you will see `always_run: false`.
However, try set `always_run: true` in `config` and do the same, you'll find `always_run: false` in `jobs`.